backend/src/main/java/com/redhat/cloud/notifications/routers/internal/HeapDumpResource.java

```diff
@@ -0,0 +1,63 @@
+package com.redhat.cloud.notifications.routers.internal;
+
+import com.sun.management.HotSpotDiagnosticMXBean;
+import io.quarkus.logging.Log;
+import jakarta.annotation.security.RolesAllowed;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.PathParam;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.core.Response;
+import org.apache.http.HttpStatus;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+import javax.management.MBeanServer;
+import java.io.File;
+import java.io.IOException;
+import java.lang.management.ManagementFactory;
+
+import static com.redhat.cloud.notifications.Constants.API_INTERNAL;
+import static com.redhat.cloud.notifications.auth.ConsoleIdentityProvider.RBAC_INTERNAL_ADMIN;
+import static jakarta.ws.rs.core.MediaType.APPLICATION_OCTET_STREAM;
+
+@Path(API_INTERNAL + "/heap_dump")
+@RolesAllowed(RBAC_INTERNAL_ADMIN)
+public class HeapDumpResource {
+
+    @ConfigProperty(name = "quarkus.log.cloudwatch.log-stream-name", defaultValue = "localhost")
+    String hostname;
+
+    @GET
+    @Path("{hostname}")
```

Comment:
Shouldn't this start with a `/`?
```suggestion
    @Path("/{hostname}")
```

Reply:
Not necessary, it works with or without it, and because they didn't added this `/` en Quarkus doc samples, I didn't added it here.